rex/utilities/loggers.py

```diff
@@ -582,3 +584,20 @@ def log_mem(logger, log_level='DEBUG'):
         logger.info(msg)
 
     return msg
+
+
+def log_versions(logger):
+    """Log package versions:
+    - rex to info
+    - h5py, numpy, pandas, and scipy to debug
+
+    Parameters
+    ----------
+    logger : logging.Logger
+        Logger object to log memory message to.
+    """
+    logger.info('Running with rex version {}'.format(__version__))
+    logger.debug('- hdf5 version {}'.format(h5py.__version__))
```

Comment:
I think this should say "h5py version {}" to match the actual package name (h5py)

Reply:
good catch!